js/projects/home.js

```diff
@@ -20,10 +20,10 @@ let loadAllProjects = async () => {
             for (let n = 0; n < json[i].access.length; n++) {
                 if (json[i].access[n].permission == "w") {
```

Comment:
wtf???

Reply:
have added span tags as seen in commit 8949364

Reply:
I will fix some css for it then